applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_node_utilities.cpp

```diff
@@ -78,4 +93,42 @@ KRATOS_TEST_CASE_IN_SUITE(NodeUtilities_UpdatesEverythingWhenNoComponentIsFixed,
     KRATOS_EXPECT_VECTOR_NEAR(expected_vector, node.FastGetSolutionStepValue(ACCELERATION, 0), 1e-12)
 }
 
+KRATOS_TEST_CASE_IN_SUITE(AssignUpdatedVectorVariableToNodes_UpdatesEverythingRegardlessOfFixes,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    auto p_node          = make_intrusive<Node>(1, 0.0, 0.0, 0.0);
+    auto nodes_container = NodeContainerWithOneFixedDOF(p_node);
+
+    // Act
+    const array_1d<double, 3> new_values{1.0, 2.0, 4.0};
+    NodeUtilities::AssignUpdatedVectorVariableToNodes(nodes_container, ACCELERATION, new_values, 0);
+    NodeUtilities::AssignUpdatedVectorVariableToNodes(nodes_container, ACCELERATION, new_values, 1);
+
+    // Assert
+    const array_1d<double, 3> expected_vector{1.0, 2.0, 4.0};
+    KRATOS_EXPECT_VECTOR_NEAR(expected_vector, p_node->FastGetSolutionStepValue(ACCELERATION, 0), 1e-12)
+    KRATOS_EXPECT_VECTOR_NEAR(expected_vector, p_node->FastGetSolutionStepValue(ACCELERATION, 1), 1e-12)
```

Comment:
I would prefer to use the default tolerance values for testing:

```suggestion
    KRATOS_EXPECT_VECTOR_NEAR(expected_vector, p_node->FastGetSolutionStepValue(ACCELERATION, 0), Defaults::absolute_tolerance)
    KRATOS_EXPECT_VECTOR_NEAR(expected_vector, p_node->FastGetSolutionStepValue(ACCELERATION, 1), Defaults::absolute_tolerance)
```

Reply:
done